internal/benthos/benthos-builder/builders/sql-util.go

```diff
@@ -906,3 +896,25 @@ func cleanPostgresType(dataType string) string {
 	}
 	return strings.TrimSpace(dataType[:parenIndex])
 }
+
+func shouldOverrideColumnDefault(columnDefaults map[string]*neosync_benthos.ColumnDefaultProperties) bool {
+	for _, cd := range columnDefaults {
+		if cd != nil && !cd.HasDefaultTransformer && cd.NeedsOverride {
+			return true
+		}
+	}
+	return false
+}
+
+func getSqlBatchProcessors(driver string, columns []string, columnDataTypes map[string]string, columnDefaultProperties map[string]*neosync_benthos.ColumnDefaultProperties) (*neosync_benthos.BatchProcessor, error) {
+	switch driver {
+	case sqlmanager_shared.PostgresDriver, "postgres":
```

Comment:
There is a new constant in the sqlmanager_shared for the postgres driver called `DefaultPostgresDriver`